### ISSUE TITLE
ProLiant MicroServer Gen8 unable to boot .

### ISSUE THREAD

**Issue by Kowalskixx** [fbelavenuto/arpl#322]
<html>
<body>

iLO 4
--
![20221116081612](https://user-images.githubusercontent.com/117470956/202052192-1e7ffdc7-cc88-41d9-940e-7de0b2d835de.png)

Decompressing Linux... Parsing ELF... Performing relocations..
Booting the kernel.

</body>
</html>

**Comment by fbelavenuto** [comment]
Please try with and without EFI enabled

**Comment by Kowalskixx** [comment]
Thanks  for your reply .I will try it shortly.

**Comment by cvc90** [comment]
> Thanks for your reply .I will try it shortly.

Have you been able to try it?

**Comment by fbelavenuto** [comment]
There is one more thing to try, enable direct boot

**Comment by fbelavenuto** [comment]
Some success?

**Comment by Kowalskixx** [comment]
Yeah dude .I think the only issue is the disk map.and  got affected COVID-19 . 😂
________________________________
发件人: Fabio Belavenuto ***@***.***>
发送时间: Monday, December 19, 2022 9:06:01 PM
收件人: fbelavenuto/arpl ***@***.***>
抄送: Kowalskixx ***@***.***>; Author ***@***.***>
主题: Re: [fbelavenuto/arpl] ProLiant MicroServer Gen8 unable to boot . (Issue #322)


Some success?

**Comment by guitengyue** [comment]
> Please try with and without EFI enabled

Encountered the same problem on Inspur server , SA5112M4
when I set the efi disable , the interface is the same as above .
when I set the efi enable , The booting interface is stuck

**Comment by ByteRipper2022** [comment]
I encouter just the same problem on my J1800 nas.

I've tried nearly every bios setting without success. The only thing that helps is removing one ram stick. I don't know why because the stick is working properly.

**Comment by we11adam** [comment]
Having trouble with HP MicroServer Gen8, too.

Host OS: PVE 7.3
VM:  8GM RAM/Q35
ARPL Config: 
<img width="1349" alt="image" src="https://user-images.githubusercontent.com/319494/210224287-ca1a9a1d-5c59-43bb-ad9b-c60c1e047169.png">


Serial port console log: https://pastebin.com/wH3nz62x

**Comment by fbelavenuto** [comment]
> Having trouble with HP MicroServer Gen8, too.
> 
> Host OS: PVE 7.3 VM: 8GM RAM/Q35 ARPL Config: <img alt="image" width="1349" src="https://user-images.githubusercontent.com/319494/210224287-ca1a9a1d-5c59-43bb-ad9b-c60c1e047169.png">
> 
> Serial port console log: https://pastebin.com/wH3nz62x

Hi, please try to enable the direct boot into advanced menu, is necessary for HP microserver

**Comment by we11adam** [comment]
> > Having trouble with HP MicroServer Gen8, too.
> > Host OS: PVE 7.3 VM: 8GM RAM/Q35 ARPL Config: <img alt="image" width="1349" src="https://user-images.githubusercontent.com/319494/210224287-ca1a9a1d-5c59-43bb-ad9b-c60c1e047169.png">
> > Serial port console log: https://pastebin.com/wH3nz62x
> 
> Hi, please try to enable the direct boot into advanced menu, is necessary for HP microserver

Thanks for reply!

However it didn't work either even if I have enabled direct boot. It boots perfectly fine with or without direct boot if I remove the PCI passthrough(for now it's a SATA controller).

**Comment by fbelavenuto** [comment]
Please update ARPL, reboot, update modules, addons and lkm, rebuild the loader and test it.

**Comment by fbelavenuto** [comment]
Maybe related #260

**Comment by we11adam** [comment]
> Please update ARPL, reboot, update modules, addons and lkm, rebuild the loader and test it.

Thanks! 👍  My server is currently occupied for something else. Will definitely try it out when I have the chance.
